.github/workflows/ci.yaml

```diff
@@ -16,10 +16,9 @@ jobs:
 
     strategy:
       matrix:
-        php-version: ['7.3', '7.4']
+        php-version: ['7.4']
         dependencies: ['']
         include:
-          - { php-version: '7.3', dependencies: '--prefer-lowest' }
```

Comment:
Might be better to leave it but change 7.3 to 7.4

Reply:
fixed, thanks!